e2e-tests/pages/shared/index.js

```diff
@@ -14,6 +14,7 @@ const body = () => cy.get('[data-cy="body"]');
 const form = () => cy.get('[data-cy="form"]');
 const heading = () => cy.get('[data-cy="heading"]');
 const headingCaption = () => cy.get('[data-cy="heading-caption"]');
+const hint = () => cy.get('[data-cy="hint"]');
```

Comment:
This is already available in `field`.

Reply:
that one has a field id before it and this hint is just data-cy hint

Reply:
Can we have the same approach as `CONNECTION_WITH_BUYER`? That has a hint displayed before the radio.

It's strange to have a `hint` like this not related to a field.